elfo-dumper/src/actor.rs

```diff
@@ -106,7 +109,10 @@ impl Dumper {
                     let config = self.ctx.config();
                     self.interval.set_period(config.write_interval);
 
-                    path = config.path(self.ctx.key());
+                    path = config.path.render(TemplateVariables {
```

Comment:
We should rerender the path on every `DumpingTick`

Reply:
Done, there's some cost in that though, but it's not like that matters here